apertium-eng.eng.dix

```diff
@@ -16642,6 +16644,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="popcorn">         <i>popcorn</i><par n="house__n"/></e>
 <e lm="pope">            <i>pope</i><par n="house__n"/></e>
 <e lm="pop">             <i>pop</i><par n="east__n"/></e>
+<e lm="pop culture">             <i>pop culture</i><par n="house__n"/></e>
```

Comment:
This should be `<i>pop<b/>culture</i>` because it's two words, see [this](http://wiki.apertium.org/wiki/Contributing_to_an_existing_pair#Adding_multiwords).

Reply:
Fixed this

Reply:
Also, the spacing should be cleaned up a bit.

Reply:
@pranavad this is still pending

Reply:
To be honest, I couldn't fix the spacing in Notepad or on GitHub. It's either getting too close or too far away

Reply:
Is it a problem if it's left like this?

Reply:
You should fix this.  Find a text editor that supports monospace fonts—this is an important skill to learn as an open-source contributor :)

Reply:
No, there can't be a space before the `<b/>`